data_extraction/requirements.txt

```diff
@@ -126,4 +126,6 @@ chromadb==0.4.22
 sseclient-py==1.8.0
 
 #Addn Code Formatting
-black
\ No newline at end of file
+black
+ruff==0.6.7
```

Comment:
Could you put 0.6.9 for ruff? I have update it in a branch, but would like the newest version everywhere.